ui/src/components/Header.vue

```diff
@@ -8,7 +8,7 @@
       <div class="nav-container">
         <b-navbar toggleable="lg" type="light" variant="faded">
           <b-navbar-nav :to="{ name: 'WorkSpacePage' }">
-            <b-nav-item class="logo" :to="{ name: 'WorkSpacePage' }">
+            <b-nav-item data-testid="ejectx-logo" class="logo" :to="{ name: 'WorkSpacePage' }">
```

Comment:
I would add the prefix "e2e" as well, e.g. data-e2e-....
data prefix can be used by frontend itself for other frontend related business logic (in the future), so adding a prefix just for e2e relevance makes it more simply to distinguish

Reply:
agree. now it would be more descriptive about its purpose.